<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1428</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Start_Button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="First_Scene">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Second_Scene">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="drum1s">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Press_To_Start_Easy">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>221</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="drum2s">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2s</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum3s">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3s</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum4s">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4s</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum5s">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5s</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum6s">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6s</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum7s">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>7s</string>
    </property>
   </widget>
   <widget class="QLabel" name="counttimer">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>90</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MisterEarl BT</family>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>counttimer</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Score">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MisterEarl BT</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>-3</number>
    </property>
   </widget>
   <widget class="QLabel" name="Final">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Restart_Button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bigi1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>410</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="bigi2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit_Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="face">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Press_To_Start_Hard">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>440</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="drum1b">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>drum1b</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum2b">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2b</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum3b">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3b</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum4b">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4b</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum5b">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5b</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum6b">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>220</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>6b</string>
    </property>
   </widget>
   <widget class="QLabel" name="drum7b">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>7b</string>
    </property>
   </widget>
   <widget class="QLabel" name="showscore">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Real_Start">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Time">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>100</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MisterEarl BT</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="finalscore">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>230</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MisterEarl BT</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Again">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SCORE">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MisterEarl BT</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>-3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="Real_Start2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="face2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>550</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sohbi1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="sohbi2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="sohbi3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="sohbi4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="cute1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>560</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cute2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>560</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cute3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cute4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Resume">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="word">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MisterEarl BT</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SCORE: </string>
    </property>
   </widget>
   <zorder>sohbi3</zorder>
   <zorder>sohbi2</zorder>
   <zorder>sohbi1</zorder>
   <zorder>sohbi4</zorder>
   <zorder>Second_Scene</zorder>
   <zorder>First_Scene</zorder>
   <zorder>bigi1</zorder>
   <zorder>bigi2</zorder>
   <zorder>counttimer</zorder>
   <zorder>Time</zorder>
   <zorder>Again</zorder>
   <zorder>drum3b</zorder>
   <zorder>drum1s</zorder>
   <zorder>drum1b</zorder>
   <zorder>drum3s</zorder>
   <zorder>drum2s</zorder>
   <zorder>drum4b</zorder>
   <zorder>drum2b</zorder>
   <zorder>SCORE</zorder>
   <zorder>drum7b</zorder>
   <zorder>drum6b</zorder>
   <zorder>drum5b</zorder>
   <zorder>drum5s</zorder>
   <zorder>drum7s</zorder>
   <zorder>drum4s</zorder>
   <zorder>drum6s</zorder>
   <zorder>face</zorder>
   <zorder>face2</zorder>
   <zorder>cute1</zorder>
   <zorder>cute2</zorder>
   <zorder>cute3</zorder>
   <zorder>cute4</zorder>
   <zorder>back_button</zorder>
   <zorder>back_button_2</zorder>
   <zorder>Pause</zorder>
   <zorder>Final</zorder>
   <zorder>Resume</zorder>
   <zorder>Score</zorder>
   <zorder>Restart_Button</zorder>
   <zorder>showscore</zorder>
   <zorder>finalscore</zorder>
   <zorder>word</zorder>
   <zorder>Press_To_Start_Hard</zorder>
   <zorder>Press_To_Start_Easy</zorder>
   <zorder>Start_Button</zorder>
   <zorder>Exit_Button</zorder>
   <zorder>Real_Start</zorder>
   <zorder>Real_Start2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1428</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
